autoflake.py

```diff
@@ -625,13 +630,19 @@ def fix_file(filename, args, standard_out):
 
     original_source = source
 
+    if args.ignore_init_module_imports and filename.endswith("__init__.py"):
+        ignore_init_module_imports = True
+    else:
+        ignore_init_module_imports = False
+
     filtered_source = fix_code(
         source,
         additional_imports=args.imports.split(',') if args.imports else None,
         expand_star_imports=args.expand_star_imports,
         remove_all_unused_imports=args.remove_all_unused_imports,
         remove_duplicate_keys=args.remove_duplicate_keys,
-        remove_unused_variables=args.remove_unused_variables)
+        remove_unused_variables=args.remove_unused_variables,
+        ignore_init_module_imports=ignore_init_module_imports)
```

Comment:
again, move the `)` to the next line, and end your line with a comma.

Reply:
Fixed!